src/server/connectors/config-db/config-db.connector.js

```diff
@@ -0,0 +1,100 @@
+const mongodb = require("mongodb");
+const { pathConfigCollectionDouble } = require("./config-db.double");
+const { CONFIGDB_URL } = require("../../config");
+const { logEmitter } = require("../../services/logging.service");
+const { statusEmitter } = require("../../services/statusEmitter.service");
+
+let client;
+let configDB;
+let pathConfigCollection;
+
+let pathConfig = null;
```

Comment:
won't this mean that we need to restart the service to get a new path config? To get around this we could force every session to pick up the "latest config" when their session starts